web/src/main/java/pl/edu/agh/rest/login/entity/LoginResultDto.java

```diff
@@ -0,0 +1,26 @@
+package pl.edu.agh.rest.login.entity;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.Optional;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableLoginResultDto.class)
+@JsonDeserialize(as = ImmutableLoginResultDto.class)
+public interface LoginResultDto {
```

Comment:
`LoginResultDTO` dla konsystencji z javowymi konwencjami

Reply:
Te interfejsy będą chyba generowane przez Swaggera, więc i tak wylecą.
@szbrnl popraw mnie jeśli się mylę

Reply:
Prawda. 

@Sopel97 mógłbyś sprawdzić czy u Ciebie działa wersja z githuba?

Reply:
PS @Sopel97 o jakich Javowych konwencjach mówisz? :open_mouth:

Reply:
zeby akronimy typu HTML, HTTP w nazwach klas nie zamienia sie na PasacalCase

Reply:
@szbrnl nie
https://pastebin.com/da7iTRNz

Reply:
@Sopel97 masz jakiś link do oficjalnego dokumentu? Z tego co pamiętam to nie było nigdzie takiej konwencji.

Wyciąg z drzewa klas JavaSE 8:
```
java.net.HttpCookie
java.net.HttpRetryException
javax.xml.ws.http.HTTPException
java.net.HttpURLConnection    <-- !!!
java.beans.XMLEncoder
javax.xml.bind.annotation.adapters.XmlAdapter
javax.xml.bind.annotation.XmlElement
javax.xml.stream.XMLEventReader
```

Reply:
@Sopel97 za pierwszym razem nie miałeś zależności, za drugim nie masz wygenerowanych klas

Reply:
dobra, poradzilem sobie z kompilacja
ale mam ten sam problem co ty, przy czym zmiana na rest/login dalej nie zmienia response

Reply:
@Sopel97 wyciąg z `man curl`:
```
       -d, --data <data>
              (HTTP) Sends the specified data in a POST request to the HTTP server, in the same way  that  a  browser
              does when a user has filled in an HTML form and presses the submit button. This will cause curl to pass
              the data to the server  using  the  content-type  application/x-www-form-urlencoded.   Compare  to  -F,
              --form.
```

Reply:
dobra, ogarnalem
`W:\123>curl -i -H "Content-type: application/json" --data-ascii "{\"username\":
\"xxx\",\"admin\": \"true\"}" -X POST localhost:8080/rest/login
HTTP/1.1 200
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Pragma: no-cache
Expires: 0
X-Frame-Options: DENY
Content-Type: application/json;charset=UTF-8
Transfer-Encoding: chunked
Date: Sat, 06 Apr 2019 13:13:02 GMT

{"valid":true,"token":"4948f7dc-de40-41be-8054-763d62b03dff"}`

Reply:
do testowania api polecam Postmana

Reply:
> > zeby akronimy typu HTML, HTTP w nazwach klas nie zamienia sie na PascalCase
> 
> @Sopel97 masz jakiś link do oficjalnego dokumentu? Z tego co pamiętam to nie było nigdzie takiej konwencji.
> 
> Wyciąg z drzewa klas JavaSE 8:
> 
> ```
> java.net.HttpCookie
> java.net.HttpRetryException
> javax.xml.ws.http.HTTPException
> java.net.HttpURLConnection    <-- !!!
> java.beans.XMLEncoder
> javax.xml.bind.annotation.adapters.XmlAdapter
> javax.xml.bind.annotation.XmlElement
> javax.xml.stream.XMLEventReader
> ```

O, widzę, że dyskusja w tej istotnej kwestii już trwa. Ja oddałam swój głos [w innym miejscu](https://github.com/papaya-agh/papaya-backend/pull/10/files#r272803805) jakby co